devtools/common/src/main/java/io/quarkus/BasicRest.java

```diff
@@ -167,8 +167,8 @@ private void createDockerFile() throws IOException {
     }
 
     private void createMicroProfileConfig() throws IOException {
-        File meta = new File(projectRoot, "src/main/resources/META-INF");
-        File file = new File(mkdirs(meta), "microprofile-config.properties");
+        File meta = new File(projectRoot, "src/main/resources");
+        File file = new File(mkdirs(meta), "application.properties");
```

Comment:
Good catch. Maybe we should rename the method to `createApplicationConfig`?

Reply:
Even better catch :P, We certainly should!

Reply:
Fixed